app/component/lrg-entry/index.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+require('./_lrg-entry.scss');
+
+const angular = require('angular');
+const ramble = angular.module('ramble');
+
+ramble.component('rambleLrgEntry', {
+  template: require('./lrg-entry.html'),
+  controller: 'LrgEntryController',
+  controllerAs: 'LrgEntryCtrl'
+});
+
+ramble.controller('LrgEntryController', ['$q', '$location', '$routeParams', 'rambleService',  LrgEntryController]);
```

Comment:
Its not an bad pratices to use shorthand like Lrg in your naming conventions, But i allways recommend using the full word because it make code easier ti read and we have tab completion anyway :)